ras_party/controllers/queries.py

```diff
@@ -582,6 +584,26 @@ def query_enrolment_by_survey_business_respondent(respondent_id, business_id, su
     return response
 
 
+def query_respondents_and_status_by_survey_and_business_id(
+    survey_id: UUID, business_id: UUID, session: session
+) -> list:
+    """
+    Query to return a list of respondents and their enrolment status in a survey for a business
+
+    :param survey_id: the uuid of the survey
+    :param business_id: the uuid of the business
+    :param session: A db session
+    :return: list of respondent ids
+    """
+
+    return (
+        session.query(Respondent, Enrolment.status)
+        .join(Respondent, Respondent.id == Enrolment.respondent_id)
+        .filter(and_(Enrolment.survey_id == survey_id, Enrolment.business_id == business_id))
+        .all()
```

Comment:
Looks ok and returns the expected data. I'm not overly familiar with the best way to form SQLAlchemy queries, or whether chaining a `.join` with a `.filter` is preferable over simply forming the column joins _within_ the filter. It doesn't appear that there's much difference and eyeballing the resulting query is good.